pkgs/applications/editors/vim/plugins/utils/update.py

```diff
@@ -1,170 +1,253 @@
-#!/usr/bin/env python
-
+#!/usr/bin/env nix-shell
+#!nix-shell -i python3 -p "python3.withPackages(ps: with ps; [ nixpkgs-updaters-library ])"
```

Comment:
I believe this is more readable:

```suggestion
#!nix-shell -i python3 -p 'python3.withPackages (ps: [ ps.nixpkgs-updaters-library ])'
```